### PR TITLE
Support column specifying

### PR DESCRIPTION
**Pull request by yoshihara**
If Jira attributes are specified in `config.yml`, `embulk run config.yml` displays specified attributes as columns. Please see `sample_config.yml`.

### CODE REVIEW

File: lib/jira/issue.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    def initialize(raw)
+      @raw = raw
+      @fields = @raw.fetch("fields")
+    end
+
+    def [](attribute)
+      case attribute
+      when "summary"
+      @fields["summary"]
```
Comment (uu59) [inline lib/jira/issue.rb]: :eyes: indent

Comment (yoshihara) [inline lib/jira/issue.rb]: OOPS X(

File: lib/jira/issue.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+module Jira
+  class Issue
+    SUPPORTED_ATTRIBUTES = ["summary", "project"].map(&:freeze).freeze
+    SUPPORTED_ATTRIBUTE_NAMES = SUPPORTED_ATTRIBUTES.map {|attr| "'#{attr}'"}.join(', ')
+
+    attr_reader :fields
+
+    def initialize(raw)
+      @raw = raw
+      @fields = @raw.fetch("fields")
```
Comment (uu59) [inline lib/jira/issue.rb]: `@raw` is only used here so unnecessary to assign as instance variable (or any plan to use this in the future?)

File: lib/jira/issue.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+module Jira
+  class Issue
+    SUPPORTED_ATTRIBUTES = ["summary", "project"].map(&:freeze).freeze
+    SUPPORTED_ATTRIBUTE_NAMES = SUPPORTED_ATTRIBUTES.map {|attr| "'#{attr}'"}.join(', ')
+
+    attr_reader :fields
+
+    def initialize(raw)
```
Comment (uu59) [inline lib/jira/issue.rb]: I'm glad if `raw` is more expositive name :)

File: lib/jira/issue.rb (hunk 2 above)
Comment (yoshihara) [inline lib/jira/issue.rb]: Oh, I have no plan to use it...

File: lib/jira/issue.rb (hunk 3 above)
Comment (yoshihara) [inline lib/jira/issue.rb]: thanks! I revise it

File: lib/embulk/input/jira.rb [hunk 4]
```diff
@@ -40,9 +41,34 @@ def self.resume(task, columns, count, &control)
       #  return {"columns" => columns}
       #end
 
+      def self.extract_attributes(attribute_names)
+        unsupported_attributes = []
+        attribute_names.each do |attribute_name|
+          unless Jira::Issue::SUPPORTED_ATTRIBUTES.include?(attribute_name)
+            unsupported_attributes << attribute_name
+          end
+        end
```
Comment (uu59) [inline lib/embulk/input/jira.rb]: Probably, this block would be simplified as below:

``` ruby
unsupported_attributes = attribute_names - Jira::Issue::SUPPORTED_ATTRIBUTES
```

File: sample_config.yml [hunk 5]
```diff
@@ -5,7 +5,10 @@ in:
   uri: 'https://treasure-data.atlassian.net'
   # oauth_token:
   # oauth_secret:
-  jql: "key=PLT-4048"
+  attributes:
+    summary: string
+    project: string
```
Comment (uu59) [inline sample_config.yml]: Also this sample should be update as an array :)

File: lib/embulk/input/jira.rb (hunk 4 above)
Comment (yoshihara) [inline lib/embulk/input/jira.rb]: I think so too :)

File: spec/jira/issue_spec.rb [hunk 6]
```diff
@@ -0,0 +1,86 @@
+require "spec_helper"
+require "jira/issue"
+
+describe Jira::Issue do
+  describe ".detect_attriute_type" do
```
Comment (uu59) [inline spec/jira/issue_spec.rb]: s/attriute/attribute/

Comment (yoshihara) [inline spec/jira/issue_spec.rb]: Oh...

File: sample_config.yml (hunk 5 above)
Comment (yoshihara) [inline sample_config.yml]: I forgot it...

**Comment by uu59** [comment]
LGTM :+1:

**Pull request merged**